src/ui/src/components/command-palette/command-text-field.tsx

```diff
@@ -132,6 +132,11 @@ const useFieldStyles = makeStyles((theme: Theme) => createStyles({
     '&:not(:first-child)': {
       borderTop: theme.palette.border.unFocused,
     },
+    '& > $optionGroupHeading + ul > li': {
```

Comment:
This is a bit incomprehensible to me, but I don't know CSS that well so maybe its common?

Reply:
Yes, this is pretty common stuff plus two Material-specific things:

- `& ...` means "prepend the rule that got us here", which is just a shorthand.
- `$anotherRule` is also a Material thing. It makes sure to copy over the generated class name correctly.
- 'foo > bar` means "a bar that is a direct child of a foo"
- `foo + bar` means "a bar that is the immediate next sibling of a foo"

All in all, this means "list items that are in groups that have headers" (as the comment implies).